Source/control.cpp

```diff
@@ -1691,11 +1692,13 @@ void ReleaseChrBtns(bool addAllStatPoints)
 	}
 }
 
-static int DrawDurIcon4Item(CelOutputBuffer out, ItemStruct *pItem, int x, int c)
+#define DURABILITY_RED 2

```

Comment:
Why are these defined in `control.cpp`? These are item things and should be moved into a more appropriate header (not CPP), like `items.h`